src/chipper/keyboard.cljs

```diff
@@ -42,13 +43,13 @@
     common-mappings
     {:edit-attr
      (merge
-       {:KeyA :C   :ShiftKeyA :C# :KeyW :C#
-        :KeyS :D   :ShiftKeyS :D# :KeyE :D#
-        :KeyD :E
-        :KeyF :F   :ShiftKeyF :F# :KeyT :F#
-        :KeyG :G   :ShiftKeyG :G# :KeyY :G#
-        :KeyH :A   :ShiftKeyH :A# :KeyU :A#
-        :KeyJ :B
+       {:KeyA 0   :ShiftKeyA 1 :KeyW 1
```

Comment:
it is kind of annoying that this just magically works because our internal reprs are ints and you don't need to import the note names, but w/e

same as before, too